packages/clay-tooltip/src/ClayTooltip.js

```diff
@@ -80,17 +80,25 @@ class ClayTooltip extends Component {
 	 */
 	_handleMouseEnter(event) {
 		const target = event.delegateTarget;
-		let title = target.getAttribute('title');
-		if (title) {
-			target.setAttribute('data-title', title);
+
+		const titleAttribute = target.getAttribute('title');
+
+		if (titleAttribute) {
+			target.setAttribute('data-title', titleAttribute);
+			target.setAttribute('data-restore-title', 'true');
 			target.removeAttribute('title');
-		} else {
-			title = target.getAttribute('data-title');
+		} else if (target.tagName === 'svg') {
+			let titleTag = target.querySelector('title');
+
+			if (titleTag) {
+				target.setAttribute('data-title', titleTag.innerHTML);
+				target.setAttribute('data-restore-title', 'true');
+				titleTag.remove();
```

Comment:
Do we really need to do this for svgs? The only reason we remove the `title` attribute is to make sure it doesn't show the default tooltip. Does it also happens for svgs?

Reply:
@jbalsas yep 😅